filters/crop/CropFilter.cpp

```diff
@@ -113,10 +115,19 @@ void CropFilter::processOptions(const Options& options)
     {
         m_geosEnvironment = initGEOS_r(pdal::geos::_GEOSWarningHandler,
             pdal::geos::_GEOSErrorHandler);
-        for (std::string& poly : m_polys)
+        for (std::string poly : m_polys)
         {
             GeomPkg g;
 
+            if (!m_assignedSRS.empty())
+            {
+                // FIXME: This should use the PointTableRef's SpatialReference
+                pdal::gdal::SpatialRef in_ref(m_assignedSRS);
+                pdal::gdal::SpatialRef out_ref(getSpatialReference().getWKT());
+                pdal::gdal::Geometry geom(poly, in_ref);
+                geom.transform(out_ref);
+                poly = geom.wkt();
+            }
```

Comment:
I would think we should also transform a bounds box.  Also, we should probably put this bit somewhere else, as I would think it's used in other places, too.